packages/iTwinUI-react/src/core/Toast/Toaster.tsx

```diff
@@ -115,8 +107,37 @@ export default class Toaster {
     this.updateView();
   }
 
+  private async createContainer() {
```

Comment:
Maybe add a comment that this is done for SSR/SSG that we wouldn't make the same mistake again.

Reply:
added